### ISSUE TITLE
"trapped" after fighting the void spider

### ISSUE THREAD

**Issue by anoobindisguise** [CleverRaven/Cataclysm-DDA#74733]
### Is your feature request related to a problem? Please describe.

This isn't exactly a bug per se but after talking with some people on the discord it seems like a bit of unintentional gameplay.
I killed the void spider and it dumped me into the "lost level". This connects to the physics/distorted lab, which itself is locked behind a hackable door in a specific microlab variant. The issue is that my character fumbled the hacking check and the terminal shut down, leaving me sealed inside the area. 
Luckily, I was able to cobble together a means of escape, but there's a good chance I could have just ended up being totally entombed with no recourse. There's essentially no indication that this can happen.

### Solution you would like.

There are a lot of viable options.
- Make the terminal never "shut down" (permanently lost) on a failed hack.
- Make the terminal on the inside of the doors not require a hack at all.
- Add a guaranteed spawn of a loaded acetylene torch and goggles inside the physics lab.

### Describe alternatives you have considered.

The void spider could just dump you outside its lair instead of the abyss, but I think that gimmick is cool and it gives a purpose to an otherwise "pointless" area.

**Comment by John-Candlebury** [comment]
Yeah this is a bug. I'll add a way to bypass the computer from the inside.

**Comment by edwardspec** [comment]
> it dumped me into the "lost level". This connects to the physics/distorted lab

Is it always connected? I've been walking in it for 15 minutes, there was nothing but emptiness in all directions.
